documentation/asciidoc/computers/configuration/configuring-networking.adoc

```diff
@@ -1,183 +1,179 @@
+----
+nmcli radio wifi
+----
 
-==== Adding the Network Details to your Raspberry Pi
+If this command returns the text "enabled", you're ready to configure a connection. If this command returns "disabled", try enabling WiFi with the following command:
 
-Open the `wpa-supplicant` configuration file in nano:
+----
+nmcli radio wifi on
+----
 
-`sudo nano /etc/wpa_supplicant/wpa_supplicant.conf`
+==== Find Networks
 
-Go to the bottom of the file and add the following:
+To scan for wireless networks, run the following command:
 
 ----
-network={
-    ssid="testing"
-    psk="testingPassword"
-}
+nmcli dev wifi list
 ----
 
-The password can be configured either as the ASCII representation, in quotes as per the example above, or as a pre-encrypted 32 byte hexadecimal number. You can use the `wpa_passphrase` utility to generate an encrypted PSK. This takes the SSID and the password, and generates the encrypted PSK. With the example from above, you can generate the PSK with `wpa_passphrase "testing"`. Then you will be asked for the password of the wireless network (in this case `testingPassword`). The output is as follows:
+You should see output similar to the following:
 
 ----
-  network={
-	  ssid="testing"
-	  #psk="testingPassword"
-	  psk=131e1e221f6e06e3911a2d11ff2fac9182665c004de85300f9cac208a6a80531
-  }
+IN-USE  BSSID              SSID            MODE   CHAN  RATE        SIGNAL  BARS  SECURITY
+        90:72:40:1B:42:05  myNetwork       Infra  132   405 Mbit/s  89      ****  WPA2
+        90:72:42:1B:78:04  myNetwork5G     Infra  11    195 Mbit/s  79      ***   WPA2
+        9C:AB:F8:88:EB:0D  Pi Towers       Infra  1     260 Mbit/s  75      ***   WPA2 802.1X
+        B4:2A:0E:64:BD:BE  Example         Infra  6     195 Mbit/s  37      **    WPA1 WPA2
 ----
 
-Note that the plain text version of the code is present, but commented out. You should delete this line from the final `wpa_supplicant` file for extra security.
+Look in the "SSID" column for the name of the network you would like to connect to. Use the SSID and a password to connect to the network.
+
+==== Connect to a Network
 
-The `wpa_passphrase` tool requires a password with between 8 and 63 characters. To use a more complex password, you can extract the content of a text file and use it as input for `wpa_passphrase`. Store the password in a text file and input it to `wpa_passphrase` by calling `wpa_passphrase "testing" < file_where_password_is_stored`. For extra security, you should delete the `file_where_password_is_stored` afterwards, so there is no plain text copy of the original password on the system.
+We'll use `nmcli` to connect to a WiFi network.
```

Comment:
Again a minor nitpick, but "We'll use `nmcli` to ..." feels a bit superfluous, when that's the program that we've already been using in the previous steps? :man_shrugging: